tests/results/DiffTest.py

```diff
@@ -626,3 +626,21 @@ def test_delete(self):
         self.uut.delete = True
         self.assertEqual(self.uut.modified, [])
         self.uut.delete = False
+
+    def test_linebreaks(self):
+        diff = Diff(['1'])
+        lines = ['1', '2', '3']
+        appended = diff._add_linebreaks(lines)
+        skip_eof = diff._generate_linebreaks(lines)
+        empty = diff._add_linebreaks([])
+        empty_skip = diff._generate_linebreaks([])
+
+        for line in skip_eof[:-1]:
+            self.assertRegex(line, '\d\n')
```

Comment:
why not asserting against a list? easier code + easier to read + less error prone I think
```
self.assertEqual(Diff._add_linebreaks(lines), ['1\n', '2\n', '3\n'])
```